contracts/contracts/coordination/Coordinator.sol

```diff
@@ -140,9 +186,11 @@ contract Coordinator is Ownable {
             newParticipant.provider = current;
             previous = current;
         }
+
+        processRitualPayment(id, providers, duration);
         
         // TODO: Include cohort fingerprint in StartRitual event?
-        emit StartRitual(id, msg.sender, providers);
+        emit StartRitual(id, ritual.authority, providers);
```

Comment:
`StartRitual`'s 2nd parameter is indexed as "`initiator`":
- Should this be `ritual.initiator`?
- Should both `initiator` and `authority` be in the event?

Reply:
Good catch. I meant `authority` here. IMO, once we introduce the authority role, the initiator is nothing more than a sponsor so there's in principle not much interest on it. In fact, once the fee is fully processed (so there's no refund), this role is not needed anymore. I would lean towards just including the authority in the event.